src/constants/contants.ts

```diff
@@ -4,3 +4,4 @@ export const COUNTRY_CREATED_MESSAGE = 'Country Created Successfully';
 export const MONGO_SERVER_ERROR = 'MongoServerError';
 export const INTERNAL_SERVER_ERROR = 'Internal Server Error';
 export const MONGOOSE_DUPLICATE_ERROR_CODE = 11000;
+export const COUNTRY_RETRIEVED_SUCCESSFUL = 'success';
```

Comment:
since the message says just "success" change the variable name to something like SUCCESS_MESSAGE. so we can use it in other places.

Reply:
Okay bro